enterprise/aibridgeproxyd/aibridgeproxyd.go

```diff
@@ -16,6 +17,14 @@ import (
 	"golang.org/x/xerrors"
 
 	"cdr.dev/slog"
+
+	"github.com/coder/aibridge"
+)
+
+// Known AI provider hosts.
+const (
+	HostAnthropic = "api.anthropic.com"
+	HostOpenAI    = "api.openai.com"
 )
```

Comment:
Not sure why these need to be consts? I only see a single usage of each?

Reply:
Yes, currently they are only used once, but kept it as constants because we most likely will need them when configuring the allowed list, so keeping this for then.